client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.tsx

```diff
@@ -26,16 +32,60 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
         );
     };
 
+    const getStreetByCity = (cityId: string) => {
+        axios.get('/addressDetails/city/' + cityId + '/streets').then(
+            result => result && result.data && setStreetsInCity(result.data.map((node: Street) => node))
```

Comment:
You can use the '?' syntax.

Reply:
why use it here?

Reply:
More readable. You can use it anyplace you want to check if an attribute is undefined.

Reply:
maya commented to use && instead of ? because it is more readable

Reply:
if you use ? you need an else case which isnt needed here

Reply:
I didn't mean ternary expression I ment for:
result?.resultData && ...

Reply:
ok my bad